message LoginReq
{
	required string username = 1;
	required string password = 2;
}

message LoginRes
{
	required bool result = 1;
	optional bool hasRole = 2;
	optional string userId = 3;
}